pkg/agent/route/route_linux.go

```diff
@@ -128,7 +128,8 @@ type Client struct {
 	nodePortsIPv6 sync.Map
 	// clusterNodeIPs stores the IPv4 of all other Nodes in the cluster
 	clusterNodeIPs sync.Map
-	// clusterNodeIP6s stores the IPv6 of all other Nodes in the cluster
+	// clusterNodeIP6s stores the IPv6 address of all other Nodes in the cluster. It is maintained but not consumed
+	// since Multicast supports IPv6.
```

Comment:
```suggestion
	// since Multicast doesn't support IPv6 yet.
```

Reply:
I would update it as `It is maintained but not consumed until Multicast supports IPv6.` It seems I missed this comment previously.